src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/ICounterPayload.cs

```diff
@@ -41,6 +41,10 @@ internal interface ICounterPayload
         /// </summary>
         string Metadata { get; }
 
-        EventType EventType { get; set; }
+        EventType EventType { get; }
+
+        bool IsMeter { get; }
+
+        int Series { get; }
```

Comment:
I was a bit torn on adding Series into the counter payloads. I found it necessary for the unification, but perhaps we can plumb this value somewhere else into CounterMonitor.